### PR TITLE
Update zsh to 5.2

### PR DESCRIPTION
**Pull request by alt219**
Update zsh from 5.0.7 to 5.2.

### CODE REVIEW

File: spk/zsh/Makefile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 SPK_NAME = zsh
-SPK_VERS = 5.0.7
-SPK_REV = 2
+SPK_VERS = 5.2
+SPK_REV = 0
```
Comment (Diaoul) [inline spk/zsh/Makefile]: This is incremental only, so 3.

**Comment by alt219** [comment]
Apologies for not using a topic branch and for getting SPK_REV wrong in the first place.  8f01ec5 (or newer) should be correct now.  If it's easier, I can close this pull request and reissue a new pull request.

**Comment by Dr-Bean** [comment]
Maybe you'd want to correct the account (name) associated with the PR.

**Comment by alt219** [comment]
Thanks, will do.

**Comment by Dr-Bean** [comment]
@alt219 Were you still planning on fixing this?

**Comment by alt219** [comment]
Do you think closing this PR and moving it to a topic branch would be a better approach?  That would be easier/better for me in the long run.

**Comment by Dr-Bean** [comment]
Technically, it's not required, but it is best-practice to do so. Using a topic branch makes it easier to keep your master branch in sync with origin. So if you'd prefer to do that, go ahead.

**Comment by alt219** [comment]
Moved to separate topic branch, see #2070.

**Pull request closed without merging**